lib/smack/SmackRep.cpp

```diff
@@ -109,6 +196,22 @@ string SmackRep::memType(unsigned r) {
   }
 }
 
+//
+// Memory type for bit-vectors
+//
+string SmackRep::memType(unsigned r, unsigned type) {
```

Comment:
The parameter name `type` is inaccurate, and `r` is no longer obvious when there are two parameters of the same type. Better names, like `reg` and `width` or `size` would be better. Furthermore, this method should be merged with `memType(unsigned)` since the case-split on whether bitvectors are used will be done inside `int_type(unsigned)`.